CHANGELOG.md

```diff
@@ -233,7 +252,8 @@ implementation (libcontainer) is *not* covered by this policy.
    cgroups at all during `runc update`). (#2994)
 
 <!-- minor releases -->
-[Unreleased]: https://github.com/opencontainers/runc/compare/v1.1.0...HEAD
+[Unreleased]: https://github.com/opencontainers/runc/compare/v1.1.1...HEAD
+[1.1.1]: https://github.com/opencontainers/runc/compare/v1.1.0...v1.1.1
```

Comment:
NIT: I would've put this in a "`1.1.z` patch release" section (and split out the "unreleased 1.1.z" list) but that doesn't really matter. I'll fix it next time I touch the changelog.

Reply:
Also I usually put the changelog bump in the same commit as the version bump, but also this doesn't matter too much.